### PR TITLE
add a troubleshooting part with debug logging

### PR DESCRIPTION
**Pull request by g3offrey**
During my investigation on https://github.com/blitz-js/blitz/pull/2792, I discovered the `DEBUG` environment variable.
I think this variable could have helped the reporter of https://github.com/blitz-js/blitz/issues/2751 to troubleshoot his issue on Blitz console.

I initiated a new section `troubleshooting` that can contains a lot of tips like that.
That could be a "must read" for future users that encounter an issue and need help debugging their Blitz projects.

Hope you'll like this 😄

### CODE REVIEW

**Comment by flybayer** [comment]
Awesome, thanks @g3offrey! Can you also add a section showing how to enabled debug in the browser? (`localStorage.debug = 'blitz:*'`)

Location in the menu looks good.

**Comment by g3offrey** [comment]
Indeed, that is a must known. 
I wasn't aware of this part, I tried on my end and added the documentation about that in my latest commit, if you want to check it out. 😄

File: app/pages/docs/troubleshooting.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- blitz:generator
+- blitz:installer
+- blitz:manifest
+- blitz:middleware
+- blitz:new
+- blitz:postinstall
+- blitz:repl
+- blitz:rpc
+- blitz:session
+- blitz:utils
```
Comment (JuanM04) [inline app/pages/docs/troubleshooting.mdx:42]: Wrap these namespaces in `` (back tick) so they appear like this: `blitz:utils`

File: app/pages/docs/troubleshooting.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+You can also have this kind of logging for the client-side part of Blitz.
+To do so simply set the local storage variable debug to `blitz:*` or any
+namespace used by Blitz and then refresh the page.
+
+```
+localStorage.setItem('debug', 'blitz:*')
+```
+
+Please also be sure, that the devtools console, has the verbose level
+enabled.
```
Comment (JuanM04) [inline app/pages/docs/troubleshooting.mdx:55]:
```suggestion
Please, be sure that the devtools console has the verbose level
enabled.
```

**Review by JuanM04 — CHANGES_REQUESTED** [review]
Awesome, thanks! Here are some suggestions:

File: app/pages/docs/troubleshooting.mdx (hunk 1 above)
Comment (g3offrey) [reply app/pages/docs/troubleshooting.mdx:42]: Sure 😄

**Comment by g3offrey** [comment]
Thank you for your feedback @JuanM04, I updated the PR. 😄

File: app/pages/docs/troubleshooting.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- `blitz:utils`
+
+### Usage in the browser {#usage-in-the-browser}
+
+You can also have this kind of logging for the client-side part of Blitz.
+To do so simply set the local storage variable debug to `blitz:*` or any
+namespace used by Blitz and then refresh the page.
+
+```
+localStorage.setItem('debug', 'blitz:*')
```
Comment (flybayer) [inline app/pages/docs/troubleshooting.mdx:51]: simpler :)
```suggestion
localStorage.debug = 'blitz:*'
```

**Review by flybayer — APPROVED** [review]

**Pull request merged**